crates/networking/discv5/src/subnet.rs

```diff
@@ -112,6 +102,20 @@ impl SyncCommitteeSubnets {
             .get(index)
             .map_err(|err| anyhow!("Subnet ID out of bounds: {err:?}"))
     }
+
+    pub fn enable_sync_committee_subnet(&mut self, subnet_id: u8) -> anyhow::Result<()> {
+        if subnet_id < SYNC_COMMITTEE_SUBNET_COUNT as u8 {
+            self.set(subnet_id as usize, true)?;
+        }
+        Ok(())
```

Comment:
```suggestion
        ensure!(subnet_id < SYNC_COMMITTEE_SUBNET_COUNT as u8);
        self.set(subnet_id as usize, true)?;
        Ok(())
```

could you make other code like this match this pattern of using ensure instead of using an if state. because if something inputs a inappropriate id ideally we throw an error